app/scripts/model/dataProxy.js

```diff
@@ -799,6 +799,7 @@ window.DataManagerForIviz = (function($, _) {
         return window.cbio.util.deepCopyObject(this.studyCasesMap);
       },
       data: {
+        studies: {},
```

Comment:
Are these changes for this PR. were is it used?

Reply:
Actually I included a fix in this pull request, it caches the study meta info instead of fetching the data everytime when we try to get display name see line 1599-1631 in dataproxy